http-client/pom.xml

```diff
@@ -41,6 +41,11 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-xml</artifactId>
+            <version>2.13.3</version>
```

Comment:
Ser vi ikkje har spesifisert versjon for dei andre jackson-avhengnadane her (sikkert styrt via dependency management ein eller annan plass), treng vi det for denne?

Reply:
Godt poeng! Fjerner versjonen 👍